pkg/template/apis/template/validation/validation.go

```diff
@@ -71,15 +70,6 @@ func ValidateTemplateInstance(templateInstance *templateapi.TemplateInstance) (a
 			allErrs = append(allErrs, field.Required(field.NewPath("spec.secret.name"), ""))
 		}
 	}
-	if templateInstance.Spec.Requester == nil {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester"), ""))
-	} else if templateInstance.Spec.Requester.Username == "" {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester.username"), ""))
```

Comment:
hm, @liggitt's comment seems to assume no one would/could ever create a templateinstance object directly w/ the Requester value already filled in.  I don't think that's a valid assumption.